src/lib/setup/resourceService.test.ts

```diff
@@ -0,0 +1,125 @@
+import {
```

Comment:
Similar to above comment to move it to `lib\azure` folder.

Reply:
accept